kml/kml_tests/gpx_tests.cpp

```diff
@@ -80,6 +88,11 @@ UNIT_TEST(Gpx_Test_Route)
   auto line = dataFromText.m_tracksData[0].m_geometry.m_lines[0];
   TEST_EQUAL(line.size(), 3, ());
   TEST_EQUAL(line[0], geometry::PointWithAltitude(mercator::FromLatLon(54.23955053156179, 24.114990234375004), 131), ());
+  auto numberOfLines = dataFromText.m_tracksData[0].m_geometry.m_lines.size();
+  TEST_EQUAL(numberOfLines, 2, ());
+  line = dataFromText.m_tracksData[0].m_geometry.m_lines[1];
+  TEST_EQUAL(line.size(), 2, ());
+  TEST_EQUAL(line[0], geometry::PointWithAltitude(mercator::FromLatLon(55.23955053156179, 25.114990234375004), 132), ());
```

Comment:
Please rewrite here:
```
auto const & lines = dataFromText.m_tracksData[0].m_geometry.m_lines;
TEST_EQUAL(lines.size(), 2, ());
{
  auto const & line = lines[0];
  ...
  TEST_EQUAL(line.back().GetAltitude(), 0, ());
}
{
  auto const & line = lines[1];
  ...
  TEST_EQUAL(line.back().GetAltitude(), 0, ());
}
```

Reply:
Thank you for checking my work
This is my first PR, I hope I read the manual about rebase amend correctly and everything I fixed is as you usualy operate